### PR TITLE
[DG] ADY-37 added url to config

### PR DESCRIPTION
**Pull request by pandemicGander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,16 @@ export MIRAKL_SDK_PASSWORD=<pass>
 export MIRAKL_ENV_URL=<miraklEnvUrl>
 export MIRAKL_API_OPERATOR_KEY=<miraklApiOperatorKey>
 export MIRAKL_API_FRONT_KEY=<miraklApiFrontKey>
+export MIRAKL_API_FRONT_KEY=<miraklApiFrontKey>
```
Comment (lancergr) [inline README.md]: duplicate?

File: src/test/java/com/adyen/mirakl/cucumber/stepdefs/AdyenApiSteps.java [hunk 2]
```diff
@@ -13,5 +14,7 @@
 
     @Given("^a configuration for the notification (.*) has been created$")
     public void aConfigurationForTheNotificationEventTypeHasBeenCreated(String eventType) throws Throwable {
+        final GetNotificationConfigurationListResponse notificationConfigurationList = adyenNotification.getNotificationConfigurationList();
+        System.out.println(notificationConfigurationList);
```
Comment (lancergr) [inline src/test/java/com/adyen/mirakl/cucumber/stepdefs/AdyenApiSteps.java]: shall we remove that from the PR?

File: README.md [hunk 3]
```diff
@@ -11,14 +11,16 @@ export MIRAKL_SDK_PASSWORD=<pass>
...
 ```
 
-Same goes for Adyen: `ADYEN_USER_NAME`, `ADYEN_PASS`, `ADYEN_ENV`, `ADYEN_APP_NAME`. Note `ADYEN_ENV` is an enum and must be either `TEST` or `LIVE` 
+Same goes for Adyen: `ADYEN_USER_NAME`, `ADYEN_PASS`, `ADYEN_ENV`, `ADYEN_APP_NAME` and `ADYEN_ENDPOINT`. 
 ```
 export ADYEN_USER_NAME=<user>
 export ADYEN_PASS=<pass>
 export ADYEN_ENV=<TEST|LIVE>
 export ADYEN_APP_NAME=<adyenAppName>
+export ADYEN_ENDPOINT=<adyenEndpoint>
```
Comment (lancergr) [inline README.md]: let's give https://cal-test.adyen.com/cal/services as example

File: README.md (hunk 1 above)
Comment (pandemicGander) [reply README.md]: sorry :( was in a rush and messed up the `README.md`, the config below is correct thought, I'll update this to match

File: README.md (hunk 3 above)
Comment (pandemicGander) [reply README.md]: ok will do

File: src/test/java/com/adyen/mirakl/cucumber/stepdefs/AdyenApiSteps.java (hunk 2 above)
Comment (pandemicGander) [reply src/test/java/com/adyen/mirakl/cucumber/stepdefs/AdyenApiSteps.java]: We can remove this, still boiler plate for Amirk but there's no need for it to be here either. I'll remove

**Comment by lancergr** [comment]
1. we can now remove ADYEN_ENV
2. I would suggest to remove ADYEN_APP_NAME and use the adyen-mirakl-connector value.. doesn't need to be configurable IMO

**Comment by pandemicGander** [comment]
@lancergr 
`ADYEN_ENV` was requested to be configurable by @rmcotton and @jedineeper. 
I first put it as a fixed value for either dev (TEST) or prod (LIVE), but during live deploy sometimes we need to use a test environment before switching to LIVE. 

`ADYEN_APP_NAME` I'll update

**Pull request merged**